CONTRIBUTING.md

```diff
@@ -10,22 +10,20 @@ Before contributing to the `posit-sdk`, ensure that the following prerequisites
 
 - Python >=3.8
 
+## Instructions
```

Comment:
Have you confirmed that these instructions work?

Reply:
I am running some tests to validate things work as expected on a forked repository.